lab-geoff/karma.conf.js

```diff
@@ -0,0 +1,26 @@
+const webpackConfig = require('./webpack.config.js');
+webpackConfig.entry = {};
+
+module.exports = function(config) {
+  config.set({
+    webpack: webpackConfig,
+    basePath: '',
+    frameworks: ['jasmine'],
+    files: [
+      'test/*.js'
+    ],
+    exclude: [
+    ],
+    preprocessors: {
+      'test/*.js': ['webpack']
+    },
+    reporters: ['progress'],
+    port: 9876,
+    colors: true,
+    logLevel: config.LOG_INFO,
+    autoWatch: true,
+    browsers: ['Chrome'],
```

Comment:
This should have a value of PhantomJS, which may be causing an issue for your test runs?